lib/dry/view/renderer.rb

```diff
@@ -4,11 +4,11 @@
 module Dry
   module View
     class Renderer
-      include Dry::Equalizer(:paths, :format, :engine)
+      include Dry::Equalizer(:dir, :root, :engines)
 
       TemplateNotFoundError = Class.new(StandardError)
 
-      attr_reader :paths, :format, :engine, :tilts
```

Comment:
Oops, I think we want to keep `:paths` both here and above. We've switched from `:root` to `:paths` so we can support multiple view template paths.